README.md

```diff
@@ -38,7 +38,9 @@ Features
   * Piping commands together
   * 100% secure (shell expansion is impossible by design)
   * Raises exceptions on errors (no more manual status code checks)
-  * allow specification of valid exit codes
+    but allows to specify which non-zero codes are still a success run
```

Comment:
I have problem with "a success" without run as it is not success. It is e.g. from grep unsuccessful run, but it is not error, it is just not foung.